wallet/wallet.go

```diff
@@ -3294,6 +3294,20 @@ func (w *Wallet) SignTransaction(tx *wire.MsgTx, hashType txscript.SigHashType,
 	return signErrors, err
 }
 
+// ErrDoubleSpend is an error returned from PublishTransaction in case the
+// published transaction failed to propagate since it was double spending a
+// confirmed transaction or a transaction in the mempool.
+type ErrDoubleSpend struct {
+	backendError error
+}
+
+// Error returns the string representation of ErrDoubleSpend.
+//
+// NOTE: Satisfies the error interface.
+func (e *ErrDoubleSpend) Error() string {
```

Comment:
Add an `Unwrap` method for our go 1.13 fam?

Reply:
Done: https://github.com/btcsuite/btcwallet/pull/647/commits/784134b4688c709780dd8b8e3cb28d3ba85b47dc